src/core/net/ip6.cpp

```diff
@@ -184,6 +187,8 @@ ThreadError Ip6::InsertMplOption(Message &aMessage, Header &aIp6Header, MessageI
 {
     ThreadError error = kThreadError_None;
 
+    otLogFuncEntry();
```

Comment:
It's great that these logs were helpful; now, please remove them, they are not helpful nor useful anymore.